config/core/configmaps/deployment.yaml

```diff
@@ -43,3 +43,7 @@ data:
 
     # List of repositories for which tag to digest resolving should be skipped
     registriesSkippingTagResolving: "ko.local,dev.local"
+
+    # ProgressDeadline is the duration we wait for the deployment to
+    # be ready before considering it failed.
+    progressDeadline: "120s"
```

Comment:
Ah the title is wrong, this isn't config-defaults 😅 

This is the right place for it though!

Reply:
Fixed the title 😆